android/app/src/androidTest/java/org/ZingoLabs/Zingo/RustFFITest.kt

```diff
@@ -18,7 +18,8 @@ object Ufvk {
 
 data class InitFromSeed (
     val seed : String,
-    val birthday : Long
+    val birthday : Long,
+    val account_index: Long
```

Comment:
This has to do with the addition of greater than initial index addresses being supported for a given seed.  The idea then is that more addresses are supported from a single seed.  Each address has a unique index.